### ISSUE TITLE
Accessibility text is not available for headers

### ISSUE THREAD

**Issue by onepunchmeg** [TablePlus/TablePlus#2951]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Version 5.3.2 (490)
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Build 490
3. The steps to reproduce this issue:
when text has been enlarged from `preferences` > `fonts & colors` the headers are not enlarged. 
i have searched the settings for something and am unable to enlarge the headers. this is an issue for those of us with accessibility needs from being visually impaired. the headers are unreadable in their default font.
Noted: If the bug is related to data, please attach an example SQL data.
